datafusion/core/src/datasource/file_format/csv.rs

```diff
@@ -771,7 +771,7 @@ mod tests {
                 "c7: Int64",
                 "c8: Int64",
                 "c9: Int64",
-                "c10: Int64",
+                "c10: Utf8",
```

Comment:
This is due to https://github.com/apache/arrow-rs/pull/6481 -- the fix for https://github.com/apache/datafusion/issues/3174 (see https://github.com/apache/datafusion/issues/3174#issuecomment-2389545252)